### PR TITLE
make lib/jekyll.rb rubocop compliant 

### PR DESCRIPTION
**Pull request by floydpraveen**
fix for one of the file listed in #4885

### CODE REVIEW

File: lib/jekyll.rb [hunk 1]
```diff
@@ -115,8 +116,8 @@ def configuration(override = Hash.new)
     # timezone - the IANA Time Zone
     #
     # Returns nothing
-    def set_timezone(timezone)
-      ENV['TZ'] = timezone
+    def add_timezone(timezone)
```
Comment (envygeeks) [inline lib/jekyll.rb]: This is unacceptable, it's an API change, please change this back to what it was. Thanks.

File: lib/jekyll.rb [hunk 2]
```diff
@@ -154,29 +155,29 @@ def sites
     def sanitized_path(base_directory, questionable_path)
       return base_directory if base_directory.eql?(questionable_path)
 
-      questionable_path.insert(0, '/') if questionable_path.start_with?('~')
+      questionable_path.insert(0, "/") if questionable_path.start_with?("~")
       clean_path = File.expand_path(questionable_path, "/")
-      clean_path.sub!(/\A\w\:\//, '/')
+      clean_path.sub!(%r!/\A\w\:\//!, "/")
```
Comment (envygeeks) [inline lib/jekyll.rb]: The `//` at the beginning and end didn't get removed, this will break the current API.

File: lib/jekyll.rb [hunk 3]
```diff
@@ -154,29 +155,29 @@ def sites
...
 
-      if clean_path.start_with?(base_directory.sub(/\A\w\:\//, '/'))
+      if clean_path.start_with?(base_directory.sub(%r!/\A\w\:\//!, "/"))
```
Comment (envygeeks) [inline lib/jekyll.rb]: The `//` at the beginning and end didn't get removed, this will break the current API.

**Comment by envygeeks** [comment]
Thanks! Just a few comments.

**Comment by floydpraveen** [comment]
Will send different PR after correcting these

**Pull request closed without merging**